plenum/server/replicas.py

```diff
@@ -0,0 +1,132 @@
+from collections import deque
+from typing import List, Generator
+
+from plenum.server.monitor import Monitor
+from plenum.server.replica import Replica
+from stp_core.common.log import getlogger
+
+logger = getlogger()
+
+MASTER_REPLICA_INDEX = 0
+
+
+class Replicas:
+
+    def __init__(self, node, monitor: Monitor):
+        # passing full node because Replica requires it
+        self._node = node
+        self._monitor = monitor
```

Comment:
Node should not be passed to `Replicas` at all.
I did it because `Replica` require it (what should be changed).
This will be updated during further refactoring.
That's why monitor is passed separately.